pkg/registry/garden/seed/storage/tableconvertor.go

```diff
@@ -39,11 +39,12 @@ func newTableConvertor() rest.TableConvertor {
 	return &convertor{
 		headers: []metav1beta1.TableColumnDefinition{
 			{Name: "Name", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["name"]},
-			{Name: "Provider (CloudProfile)", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["cloudprofile"]},
+			{Name: "Status", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["status"]},
+			{Name: "Provider (CloudProfile)", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["provider"]},
 			{Name: "Region", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["region"]},
-			{Name: "Ingress Domain", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["domain"]},
-			{Name: "Available", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["available"]},
 			{Name: "Age", Type: "date", Description: swaggerMetadataDescriptions["creationTimestamp"]},
```

Comment:
Can we make `Age` the last column? And also can we clean up #119 as we now use this tableconvertor?

Reply:
I followed `kubectl get node`:

```yaml
NAME             STATUS   ROLES    AGE   VERSION
docker-desktop   Ready    master   38d   v1.14.7
```